app/utils/FileProblemHelper.scala

```diff
@@ -17,16 +17,26 @@
 package utils
 
 import models.fileDetails.FileErrorCode.fileErrorCodesForProblemStatus
-import models.fileDetails.RecordErrorCode.DocRefIDFormat
-import models.fileDetails.{FileErrorCode, RecordErrorCode, ValidationErrors}
+import models.fileDetails.RecordErrorCode.{CustomError, DocRefIDFormat}
+import models.fileDetails.{FileErrorCode, RecordError, RecordErrorCode, ValidationErrors}
+import viewmodels.FileRejectedViewModel.errorList
 
 object FileProblemHelper {
 
   private val expectedErrorCodes: Seq[String]       = FileErrorCode.values.map(_.code) ++ RecordErrorCode.values.map(_.code)
   private val problemsStatusErrorCodes: Seq[String] = fileErrorCodesForProblemStatus.map(_.code) :+ DocRefIDFormat.code
 
   def isProblemStatus(errors: ValidationErrors): Boolean = {
-    val errorCodes = Seq(errors.fileError.map(_.map(_.code.code)).getOrElse(Nil), errors.recordError.map(_.map(_.code.code)).getOrElse(Nil)).flatten
-    errorCodes.exists(!expectedErrorCodes.contains(_) || errorCodes.exists(problemsStatusErrorCodes.contains(_)))
+    val errorCodes: Seq[String] =
+      Seq(errors.fileError.map(_.map(_.code.code)).getOrElse(Nil), errors.recordError.map(_.map(_.code.code)).getOrElse(Nil)).flatten
+
+    errorCodes.exists(!expectedErrorCodes.contains(_) || errorCodes.exists(problemsStatusErrorCodes.contains(_))) || errorDetailNotAllowed(errors.recordError)
   }
+
+  private def errorDetailNotAllowed(errors: Option[Seq[RecordError]]): Boolean =
```

Comment:
Is it worth adding a scenario to cover this in the FileProblemHelperSpec ?

Reply:
@soljeggers It is covered in fileStatusViewModelSpec